index.js

```diff
@@ -35,19 +35,35 @@ async function run () {
     const credentialsType = await getTypeOfCredentials().catch(() => {
       console.log('An error occurred while trying to determine if production or sandbox credentials were used for ServiceNow.')
       console.log('So the link(s) provided below will be for the production environment, even though you may have used sandbox credentials. 🤷')
-      console.log('The standard change will still be started in the correct environment.')
+      console.log('The standard change will still be started in the correct environment.\n')
       return 'PRODUCTION'
     })
     const servicenowHost = (credentialsType === 'PRODUCTION') ? 'support.byu.edu' : 'support-test.byu.edu'
 
+    const alreadyCreatedRfc = await getRfcIfAlreadyCreated(linkToCommits).catch(() => {
+      warning('An error occurred while trying to determine if an RFC was already created by a previous run of this workflow.')
+      console.log('We will create a new RFC. If there was an existing RFC that failed, it will be your responsibility to update its status as appropriate.\n')
+    })
+    if (alreadyCreatedRfc) {
+      warning('An existing RFC was found!')
+      console.log(`RFC Number: ${alreadyCreatedRfc.number}
+Link to RFC: https://${servicenowHost}/change_request.do?sysparm_query=number=${alreadyCreatedRfc.number}
+Created on: ${alreadyCreatedRfc.sys_created_on}
+Last updated on: ${alreadyCreatedRfc.sys_updated_on}`)
+      // Set outputs for GitHub Actions
+      setOutput('change-sys-id', alreadyCreatedRfc.sys_id)
+      setOutput('work-start', convertServicenowTimestampFromMountainToUtc(alreadyCreatedRfc.work_start))
```

Comment:
I'm having a hard time telling if we need `convertServicenowTimestampFromMountainToUtc()` in this case, due to some existing bad data (#29, which was recently fixed). I actually suspect that we _don't_ need it here.

Reply:
I'm not familiar enough on the reasoning for this function to give a recommendation